### ISSUE TITLE
Specify specific station in mopidy config

### ISSUE THREAD

**Issue by anvarit** [mopidy/mopidy-dirble#10]
Hi, would it be possible to add one or two specific stations. This would be handy for a RaspberryPi so that everything goes much smoother. 

The Dirble api call is: GET http://api.dirble.com/v2/station/<ID>

**Comment by adamcik** [comment]
Not sure I follow what you mean by this, could you please elaborate?

**Comment by anvarit** [comment]
with the current config you can specify a country and if you open a web interface (moped, ...) you can see all the stations in that country.
This is quite resource intensive for a Pi. 

The question is if it possible to specify a specific station in the mopidy settings so when you open a webinterface you only see the stations you specified in the mopidy settings and not the whole list of stations in that country.

**Comment by adamcik** [comment]
Right, so essentially just adding some favorites directly in the config.

As for the resource intensive, do you mean displaying the web UI (are you running the browser on the PI) or do you mean intensive for Mopidy running on the PI fetching the data?

**Comment by anvarit** [comment]
Yes indeed,

It is a full install, the Rpi has a 2.8” touchscreen with a mopidy module that loads a browser on the touchscreen.
